tests/contrib/asyncio/test_tracer_safety.py

```diff
@@ -1,10 +1,14 @@
 import asyncio
 
-from ddtrace.provider import DefaultContextProvider
+import pytest
+
+from ddtrace.compat import CONTEXTVARS_IS_AVAILABLE
 from ddtrace.contrib.asyncio.compat import asyncio_current_task
+from ddtrace.provider import DefaultContextProvider
 from .utils import AsyncioTestCase, mark_asyncio
 
 
+@pytest.mark.skipif(CONTEXTVARS_IS_AVAILABLE, reason="No configuration is necessary when contextvars available.")
```

Comment:
I don't really get the reason here and why we skip this.

Reply:
Yeah so these tests were written (pre-contextvars) to ensure that when a tracer was not configured with an `AsyncioContextProvider` that, even though the traces would be wrong, they would still finish and not result in memory leaks.

Since the introduction of contextvars and `DefaultContextProvider` which uses them, this is a non-issue in asyncio since now default supports asyncio.

Lemme know if that makes sense 😆. I can add a comment in the code as well elaborating a bit more.

Reply:
Yeah if you can write that as a comment that'd be perfect, especially because there's history into that that a new reader might not be aware.